### PR TITLE
feat(menuSelect): implement is.css

### PR DESCRIPTION
**Pull request by bobylito**
[spec](https://instantsearch-css.netlify.com/widgets/menu-select/)

Nothing special about this one. Just a markup update and `attributeName` -> `attribute`

### CODE REVIEW

File: src/components/MenuSelect.js [hunk 1]
```diff
@@ -27,27 +26,29 @@ class MenuSelect extends Component {
...
-        ))}
-      </select>
+
+          {items.map(item => (
+            <option
+              key={item.value}
+              value={item.value}
+              className={cssClasses.option}
+            >
+              <Template data={item} templateKey="item" {...templateProps} />
```
Comment (bobylito) [inline src/components/MenuSelect.js]: Use template for option

File: src/widgets/menu-select/menu-select.js [hunk 2]
```diff
@@ -5,18 +5,15 @@ import connectMenu from '../../connectors/menu/connectMenu';
...
-  bemHelper,
-  prepareTemplateProps,
-  getContainerNode,
-} from '../../lib/utils';
+import { prepareTemplateProps, getContainerNode } from '../../lib/utils';
 
-const bem = bemHelper('ais-menu-select');
+import { component } from '../../lib/suit';
+
+const suit = component('menuSelect');
```
Comment (bobylito) [inline src/widgets/menu-select/menu-select.js]: wrong case, should be `MenuSelect`

File: docgen/src/guides/v3-migration.md [hunk 3]
```diff
@@ -69,6 +69,35 @@ InstantSearch 3 introduces some breaking changes in the widget's naming, options
...
+
+| Before          | After       |
+| --------------- | ----------- |
+| `attributeName` | `attribute` |
+
+#### CSS classes
+
+| Before                    | After                   |
+| ------------------------- | ----------------------- |
+| `ais-menu-select`         | `ais-MenuSelect`        |
```
Comment (francoischalifour) [inline docgen/src/guides/v3-migration.md]: The new `ais-MenuSelect--noRefinement` is missing.

File: src/components/MenuSelect.js [hunk 4]
```diff
@@ -27,27 +26,39 @@ class MenuSelect extends Component {
...
     return (
-      <select
-        className={cssClasses.select}
-        value={selectedValue}
-        onChange={this.handleSelectChange}
-      >
-        <option value="" className={cssClasses.option}>
-          <Template templateKey="seeAllOption" {...templateProps} />
-        </option>
+      <div className={cssClasses.root}>
```
Comment (francoischalifour) [inline src/components/MenuSelect.js]: Are we using [`classnames`](https://github.com/JedWatson/classnames) everywhere?

File: src/components/MenuSelect.js [hunk 5]
```diff
@@ -27,27 +26,39 @@ class MenuSelect extends Component {
...
+        <select
+          className={cssClasses.select}
+          value={selectedValue}
+          onChange={this.handleSelectChange}
+        >
+          <Template
+            templateKey="seeAllOption"
```
Comment (francoischalifour) [inline src/components/MenuSelect.js]: Let's change it to `seeAllOptions`.

File: src/components/MenuSelect.js [hunk 6]
```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'preact-compat';
 import PropTypes from 'prop-types';
+import cx from 'classnames';
 
 import Template from './Template';
 
+
```
Comment (francoischalifour) [inline src/components/MenuSelect.js]: Did you run Prettier? Seems like there's a double line here.

**Pull request merged**